packages/terra-icon/src/IconBaseDec.jsx

```diff
@@ -50,7 +50,6 @@ const IconBase = ({
   children,
```

Comment:
We should alphabetize the props in this file to be consistent. It's out of the scope of this PR and should have been updated on the [original creation PR](https://github.com/cerner/terra-core/pull/3602/files#diff-4c928a6653a1808a855a6620e9d9c72f4215734657cf63e4fad2b75520e65f26).

Reply:
I plan to update one of the icon examples so that it's more clear; i can do this as part of that PR.